README.md

```diff
@@ -115,6 +107,14 @@ From this point on, we'll assume that your web server is available at this local
 
 http://localhost:8000/
 
+Following are the routes to access our apis.
```

Comment:
@bhavya-shukla-lrn Being an acronym, "api" should always be in capitals, like this: "API".